.github/workflows/require-label.yaml

```diff
@@ -0,0 +1,37 @@
+name: require-label
+
+on:
+  workflow_call:
+    inputs:
+      label:
+        required: true
+        type: string
+    outputs:
+      result:
+        value: ${{ jobs.require-label.outputs.result }}
+
+jobs:
+  require-label:
+    runs-on: ubuntu-22.04
```

Comment:
Maybe we could use ubuntu-latest for this if there are no particular reasons for using ubuntu-22.04

Reply:
Done.